lib/projectPreprocessor.js

```diff
@@ -587,9 +587,12 @@ class ProjectPreprocessor {
 		const taskRepository = require("@ui5/builder").tasks.taskRepository;
 
 		const taskPath = path.join(extension.path, extension.task.path);
```

Comment:
`extension.path` is already absolute, right?

Reply:
Yes